README.md

```diff
@@ -14,6 +14,14 @@ The [contributing documentation](CONTRIBUTING.md) covers licencing and the usual
 
 **Do Not Ever Update Quay Labels Based On Your Local Development Branch. Only from master!**
 
+### Release names
+
+When [adding releases to channels](#add-releases-to-channels) or [blocking edges](#block-edges), releases are given by name.
```

Comment:
s/releases are given by name/releases must be given by name/

Reply:
> s/releases are given by name/releases must be given by name/

This section is really about "when you put some characters in a release-name field, how are they interpreted", not about declaring "this is a release name field".  I've pushed 34cfee3 -> 572e1fe moving to:

```markdown
Release names are used for [adding releases to channels](#add-releases-to-channels) and [blocking edges](#block-edges).
```

to make this a bit more concise "here's why you care about this section".  How do you feel about that?